Rename WebExtension provider to 'webext' (#4)
ipfs-shipyard/ipfs-provider: README.md

This module tries to connect to IPFS via multiple providers, in order:

> - `webext` looks for an instance in the [background page of a WebExtension](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/extension/getBackgroundPage) (not used on regular web).

Reviewer: What do you mean by `(not used on regular web)`?

Author: Probed API is only in WebExtensions. Does below sound better?
```suggestion
- `webext` looks for an instance in the [background page of a WebExtension](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/extension/getBackgroundPage) (used only in browser extensions, not regular pages).
```
